File: LLM-Translations/GPT-4/codenet/C/Python/s497487985.py
def gcd(a, b):
    if b == 0:
        return a
    else:
        return gcd(b, a % b)

def lcm(a, b):
    return a * b // gcd(a, b)

def main():
    n, k, q = map(int, input().split())
    a = [0] * 100001
    point = [0] * 100001

    for i in range(q):
        a[i] = int(input())
        point[a[i] - 1] += 1

    for i in range(n):
        if k - (q - point[i]) > 0:
            print("Yes")
        else:
            print("No")

if __name__ == "__main__":
    main()

File: LLM-Translations/GPT-4/codenet/C/Java/s497487985.java

import java.util.Scanner;

public class s497487985 {
    public static long gcd(long a, long b) {
        if (b == 0) {
            return a;
        } else {
            return gcd(b, a % b);
        }
    }

    public static long lcm(long a, long b) {
        return a * b / gcd(a, b);
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        long n = sc.nextLong();
        long k = sc.nextLong();
        long q = sc.nextLong();

        long[] a = new long[100001];
        long[] point = new long[100001];

        for (long i = 0; i < q; i++) {
            a[(int) i] = sc.nextLong();
            point[(int) (a[(int) i] - 1)]++;
        }

        for (long i = 0; i < n; i++) {
            if (k - (q - point[(int) i]) > 0) {
                System.out.println("Yes");
            } else {
                System.out.println("No");
            }
        }

        sc.close();
    }
}

// 
